uaclient/http/__init__.py

```
@@ -362,10 +385,12 @@ def readurl(
     else:
         resp = _readurl_urllib(req, timeout=timeout)
 
+    decoded_body = resp.body.decode("utf-8")
```

Let's just ignore errors in this decode call